scripts/test/watch

```diff
@@ -1,12 +0,0 @@
-#!/bin/sh
-
-set -e
-
-filewatcher \
-    -L 5 \
-    -x '**/*.swp' \
-    -x .git \
-    -x build \
-    -x .idea \
-    -- \
-    sh -c 'go test -timeout 10s -v ./${dir} || ( echo; echo; exit 1 )'
```

Comment:
I agree. We don't need to have it in the build system. If this becomes a burden we can remove it.

However I think there are advantages to providing something like this. It means that new developers can get started without having to create it themselves. I don't really believe that including it is a burden on anyone at this time.